src/ApiPlatform/Refresh/Message/Refresh.php

```diff
@@ -72,5 +72,5 @@ class Refresh
     /**
      * @var string
      */
-    public $refreshToken;
+    public $refresh_token;
```

Comment:
https://symfonycasts.com/screencast/api-platform/serialization-tricks#controlling-field-names-serializedname

Maybe this was also possible?